src/Symfony/Component/Notifier/Bridge/AmazonSns/AmazonSnsTransportFactory.php

```diff
@@ -31,8 +31,10 @@ public function create(Dsn $dsn): AmazonSnsTransport
 
         $host = 'default' === $dsn->getHost() ? null : $dsn->getHost();
         $port = $dsn->getPort();
+        $sslmode = $dsn->getOption('sslmode', true);
```

Comment:
Can we be consistent with SQS? It seems that we are using `disable` there to disable SSL.